src/main/java/io/github/thebusybiscuit/hotbarpets/HotbarPetsRunnable.java

```diff
@@ -29,25 +29,19 @@ public void run() {
             for (int i = 0; i < 9; ++i) {
                 ItemStack item = p.getInventory().getItem(i);
 
-                if (item == null || item.getType() == null || item.getType() == Material.AIR) {
+                if (item == null || item.getType() == Material.AIR) {
                     continue;
                 }
 
-                if (isPet(item, chicken)) {
-                    if (chicken.checkAndConsumeFood(p)) {
-                        p.getInventory().addItem(new ItemStack(Material.EGG));
-                        p.getWorld().playSound(p.getLocation(), Sound.ENTITY_CHICKEN_EGG, 1.0F, 2.0F);
-                    }
-                } else if (isPet(item, mooshroom)) {
-                    if (mooshroom.checkAndConsumeFood(p)) {
-                        p.getInventory().addItem(new ItemStack(Material.MUSHROOM_STEM));
-                        p.getWorld().playSound(p.getLocation(), Sound.ENTITY_COW_AMBIENT, 1.0F, 2.0F);
-                    }
-                } else if (isPet(item, fish)) {
-                    if (fish.checkAndConsumeFood(p)) {
-                        p.getInventory().addItem(new ItemStack(Material.COOKED_COD));
-                        p.getWorld().playSound(p.getLocation(), Sound.BLOCK_WATER_AMBIENT, 1.0F, 2.0F);
-                    }
+                if (isPet(item, chicken) && chicken.checkAndConsumeFood(p)) {
+                    p.getInventory().addItem(new ItemStack(Material.EGG));
+                    p.getWorld().playSound(p.getLocation(), Sound.ENTITY_CHICKEN_EGG, 1.0F, 2.0F);
+                } else if (isPet(item, mooshroom) && mooshroom.checkAndConsumeFood(p)) {
+                    p.getInventory().addItem(new ItemStack(Material.MUSHROOM_STEW));
+                    p.getWorld().playSound(p.getLocation(), Sound.ENTITY_COW_AMBIENT, 1.0F, 2.0F);
+                } else if (isPet(item, fish) && fish.checkAndConsumeFood(p)) {
+                    p.getInventory().addItem(new ItemStack(Material.COOKED_COD));
+                    p.getWorld().playSound(p.getLocation(), Sound.BLOCK_WATER_AMBIENT, 1.0F, 2.0F);
                 } else if (isPet(item, goldenCow) && goldenCow.checkAndConsumeFood(p)) {
```

Comment:
This is a big downgrade performance-wise.

If the item **is** a Chicken Pet but we don't have any food, we don't need to check if it is a mooshroom or a fish since we already know it is a Chicken.